components/alert/alert.css

```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+*
+{
+    --error-color:#ef4444;
+    --success-color:#15803d;
+    --warning-color:#f59e0b;
+    --info-color:#06b6d4;
```

Comment:
Suggestion:
Please avoid using color in variable names.

Reply:
Yes, writing color would be more verbose. Avoid it.

Reply:
Noted.